src/unix/pwrite.c

```diff
@@ -4,7 +4,26 @@
 #include <caml/signals.h>
 #include <caml/unixsupport.h>
 
-CAMLprim value caml_index_pwrite
+CAMLprim value caml_index_pwrite_int
```

Comment:
I tried to use macros to avoid this duplication of the bindings, but it doesn't seem to be worth it.